### PR TITLE
Lint recipe for v2.20.0

### PR DESCRIPTION
**Pull request by marcoesters**
Needed for anaconda-linter.

# Package Info

Home/doc URL: https://pre-commit.com/
Dev URL: https://github.com/pre-commit/pre-commit

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +34,7 @@ outputs:
         - python >=3.7
         - pyyaml >=5.1
         - toml
-        - virtualenv >=20.0.8
```
Comment (cbouss) [inline recipe/meta.yaml:38]: This should still be 20.0.8. See: https://github.com/pre-commit/pre-commit/blob/v2.20.0/setup.cfg#L31

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,7 +34,7 @@ outputs:
         - python >=3.7
```
Comment (cbouss) [inline recipe/meta.yaml]: python pinning should be removed, since this is not a noarch package

**Review by cbouss — COMMENTED** [review]
LGTM, if comments are addressed and abs.yaml removed/commented.

**Review by varlackc — APPROVED** [review]
We might need to skip `s390x` or correct it other than that It looks good.

File: recipe/meta.yaml (hunk 1 above)
Comment (marcoesters) [reply recipe/meta.yaml:38]: Ah, I think I looked at the main branch. Thank you!

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**